utils/release/bump-package.mjs

```diff
@@ -82,7 +83,10 @@ await (async () => {
   );
   const versionPrefix = `${packageJson.name}@`;
   const convention = await changelogConvention();
-  const lastTag = await getLastTag(versionPrefix);
+  const lastTag = await getLastTag({
```

Comment:
@louis-bompart 

Also, just to validate my understanding (for the postmortem):

Is the root cause that we're on the master branch when calling `getLastTag` and hence the function returns the last v3 tag?

If so, am I right to assume that the `getCommits(PATH, lastTag);` function call would list the commits since the last v3 tag until HEAD (on master)? I.e., all commits since the last v3 release?

This would mean that the information resolved by the bump-package script for a v2 release would have been completely faulty: it would have a the next v3 version number, and its changelog would also describe the v3 changes.

This would also explain why it worked as long as the last version tag on master was a v2 tag.

Reply:
I'm not entirely sure how `git describe --tags—-match=$packageName@` should have behaved with v2 once it started diverging from v3.

From my experiment, however, it did go back and try to fetch the v3 tags (before I rewrite the git history of master & v2).

What I know is that the exact-match offers more control as it limits the function range to only the given match (i.e. the release branch), which serves us well.